rcparse.go

```diff
@@ -353,12 +354,12 @@ func loadManifestResource(manifestFileName string) (*Resource, error) {
 	}
 	return &Resource{
 		Type: gowin32.ResourceTypeManifest,
-		Id:   1,
+		Id:   uint(wrappers.CREATEPROCESS_MANIFEST_RESOURCE_ID),
```

Comment:
I'm going to say beyond the scope of this pull request, since it's basically a bug fix.  We can always do it later as its own ticket or if we're adding more resource types and it becomes an issue.